examples/simpleGRPC.go

```diff
@@ -0,0 +1,100 @@
+package main
+
+import (
+    "context"
+    //"http"
+    "net/http"
+    "log"
+    "time"
+    "google.golang.org/grpc"
+    pb "github.com/alonsopf/sil"
```

Comment:
When I was looking at samples, I was thinking about something similar to grpc-go's model. Take a look at some of those, but the idea is that you have a `server.go` and a `client.go` file that demonstrates both sides (if needed). In this case, we really only need the server sides as you shouldn't chain system health checks together. https://github.com/grpc/grpc-go

Reply:
sorry for my errors 😔, and thanks for your patience 😀, what do you think about this sample https://pastebin.com/QL1SwLEt ?
is very similar, but use gRPC core implementations, but used client, so i should use somewhere  s.NewServer() right?

Reply:
We could do something like use the provided health client to call the API directly using the unary call. We'll want to avoid using the server streaming call as that would be bound to a single instance.

Reply:
Actually, your pastebin looks great. Let's use the hello world example as linked. You will need to run fmt again. go-groups will regroup your imports for this file.

Reply:
Thank you, i learned a few things